src/components/SpaceContainer.js

```diff
@@ -1,21 +1,26 @@
 import React, { useEffect, useState } from "react";
+import SpaceCard from "./SpaceCard";
 import axios from "axios";
-import PhotoContent from "./PhotoContent";
 
-export default function DailyPhoto( { limit }) {
-  const [photo, setPhoto] = useState([]);
+
+export default function SpaceContainer({ limit }) {
+  const [data, setData] = useState([]);
 
   useEffect(() => {
     axios
       .get("https://api.nasa.gov/planetary/apod?api_key=IpSWIoboMnzGCH5JDIFqiGhn16zuwo4kIaBJnHgi")
       .then(response => {
-        const data = response.data;
```

Comment:
even just commenting out things that you thought would work is good. this way you can see the steps you took to get to a functioning app